packages/build/src/log/header.ts

```diff
@@ -8,8 +8,7 @@ export const getHeader = function (message: string) {
   const messageWidth = stringWidth(message)
   const headerWidth = Math.max(HEADER_MIN_WIDTH, messageWidth)
   const line = '─'.repeat(headerWidth + PADDING_WIDTH * 2)
```

Comment:
```suggestion
  const line = '─'.repeat(headerWidth + PADDING_WIDTH)
```

The left padding is not there anymore, so we can reduce the whole line by one padding, so that the lines are the same length again, which they are not right now, as can be seen in the snapshot.